tests/testthat/test-cube-residuals-zed-scores.R

```diff
@@ -182,74 +182,55 @@ test_that("residuals for catarray", {
 })
 
 test_that("compareCols()", {
-    expected_zScores <- cubify(
-        2.54925480834223,
-        -2.54925480834223,
-        dims = list(
-            Gender = c("Male", "Female"),
-            RespondentIdeology = c("Very Conservative")
-        )
-    )
+    expected_chisq = structure(5.74120651376478, .Names = "X-squared")
     expect_equal(
         compareCols(
             gender_x_ideology,
             baseline = "Very liberal",
             x = "Very Conservative"
         ),
-        expected_zScores
+        expected_chisq
```

Comment:
Am I reading this right that we're changing the behavior of `compareCols()` from returning z-scores for each cell (like `zScores()` does) to returning the return value from `chisq.test`? Is there any reason not to include `stdres` in the allowed `value` parameters so that it can return what we have been already returning? 

My suspicion from talking to a few of the offices about this is that the p-value derived from those z-scores are what people are thinking they want when they compare one column against another (and the same in the pair-wise compare all columns case).

Reply:
Also, if we are transitioning away from z-scores to just providing the test statistic we should match that behavior elsewhere. I'm not certain we actually want to make that transition, however.